e

```diff
@@ -0,0 +1,256 @@
+[33mcommit 091c5dc1783effac224ab3639ddba757b24fa8b1[m[33m ([m[1;36mHEAD -> [m[1;32mcontact[m[33m, [m[1;31morigin/main[m[33m, [m[1;31morigin/contact[m[33m, [m[1;31morigin/HEAD[m[33m, [m[1;32mmain[m[33m)[m
```

Comment:
Remove this trash filel

Reply:
And Also why your changes are affecting the submodule. Please look into the issue properly.